deep-learning/_posts/2018-07-05-Ship-wake-detection.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: review
+title:  "Ship wake-detection procedure using conjugate gradient trained artificial neural networks"
+tags:   deep-learning, remote-sensing, neural-network
+author: Charles Authier
+pdf:  https://ieeexplore.ieee.org/stamp/stamp.jsp?tp=&arnumber=83986
+cite:
+  authors: "J.P. Fitch, S.K. Lehman, F.U. Dowla, S.Y. Lu, E.M. Johansson, D.M. Goodman"
+  title:   "Ship wake-detection procedure using conjugate gradient trained artificial neural networks"
+  venue:   " IEEE Transactions on Geoscience and Remote Sensing ( Volume: 29, Issue: 5, Sep 1991 )"
+---
+
+This paper developed a method to reduce large two-dimensional images to significantly smaller “feature lists.” These feature lists overcome the problem of storing and manipulating large amounts of data.
+
+A new artificial neural network using conjugate gradient training methods was successfully trained to determine the presence or absence of wakes in radar images.
+
+**Images: radar, 4000 x 4000 pixels for a 25m of resolution by pixel**
+
+Because radar satellites refresh only few seconds and with a bad image resolution, is hard to get ocean-surface features. However, the internal wave signatures due to tidal flows over the continental shelf and ship wakes were observed in Seasat data. These activities have a long duration and allow ship detection.
```

Comment:
satellite frame rate is less than a frame per second and their image resolution is low, it is hard

signature